imap_processing/lo/l1a/lo_l1a_write_cdfs.py

```diff
@@ -0,0 +1,123 @@
+"""Write Lo L1A CDFs."""
+
+import numpy as np
+import xarray as xr
+
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.lo.l0.lo_apid import LoAPID
+from imap_processing.lo.l1a import lo_cdf_attrs
+from imap_processing.lo.l1a.lo_data_container import LoContainer
+
+
+def write_lo_l1a_cdfs(data: LoContainer):
+    """
+    Write the Lo L1a CDFs.
+
+    Parameters
+    ----------
+    data : LoContainer
+        The Lo data container contaings all available Lo dataclass objects
+        for one pointing.
+    """
+    created_filepaths = []
+
+    # Write Science Direct Events CDF if available
+    science_direct_events = data.filter_apid(LoAPID.ILO_SCI_DE.value)
+    if science_direct_events:
+        scide_dataset = create_lo_scide_dataset(science_direct_events)
+        cdf_file = write_cdf(scide_dataset)
+        created_filepaths.append(cdf_file)
+
+    # TODO: Add the rest of the APIDS
+
+    return created_filepaths
+
+
+def create_lo_scide_dataset(scide: list):
+    """
+    Create Lo L1A Science Direct Event Dataset from the ScienceDirectEvent dataclasses.
+
+    Parameters
+    ----------
+    scide: list
```

Comment:
```suggestion
    sci_de: list
```

To be consistent with how you are naming the other variables in this function?